pkg/kubernetes/app.js

```diff
@@ -106,5 +144,34 @@ define([
 
         .factory('kubernetesClient', function() {
             return kubernetes.k8client();
-        });
+        })
+
+        .factory('kubernetesFilter', ['kubernetesClient',
```

Comment:
I've started using a consistent indentation form like this:

```
.factory('kubernetesFilter', [
    'kubernetesClient',
    '$location',
    function(client, $location) {
        var selected_namespace = null;
        ...
    }
]);
```

What do you think?